fixtures/topics.yaml

```diff
@@ -437,7 +437,7 @@ topics:
     -
         slug: funding-grants-rebates
         theme: government-business
-        text_en: Fudning, grants, & rebates
+        text_en: Funding, grants, & rebates
```

Comment:
you can spell!

Reply:
It was a requirement of joining the content team